src/core/qgsfeaturerequest.h

```diff
@@ -636,6 +636,40 @@ class CORE_EXPORT QgsFeatureRequest
      */
     void setConnectionTimeout( int connectionTimeout );
 
+    /**
+     * The amount of free connections required to start this request.
+     * The system will block the request until the specified amount of connections
+     * is available for usage.
+     *
+     * By default this amount is 3. This makes sure, that we have 2 spare connections
+     * that might be used by "nested" requests which are executed while iterating
+     * over the results of this request.
+     *
+     * This number should be changed to one, when we know that no nested requests happen
+     * and that this request might happen in a nested way. This is for example given for
```

Comment:
"and that this request might happen" -> "and that this request **itself** might happen "